code/src/java/pcgen/gui2/csheet/CharacterSheetPanel.java

```diff
@@ -56,61 +60,36 @@
  */
 public class CharacterSheetPanel extends HtmlPanel implements CharacterSelectionListener
 {
-
-	private static final String COLOR_TAG = "preview_color.css";
-
 	private enum CssColor
 	{
-
-		BLUE,
-		LIGHTBLUE,
-		GREEN,
-		LIGHTGREEN,
-		RED,
-		LIGHTRED,
-		YELLOW,
-		LIGHTYELLOW,
-		GREY,
-		LIGHTGREY;
-
-		public String getCssText()
-		{
-			switch (this)
-			{
-				case BLUE:
-					return "preview_color_blue.css";
-				case LIGHTBLUE:
-					return "preview_color_light_blue.css";
-				case GREEN:
-					return "preview_color_green.css";
-				case LIGHTGREEN:
-					return "preview_color_light_green.css";
-				case RED:
-					return "preview_color_red.css";
-				case LIGHTRED:
-					return "preview_color_light_red.css";
-				case YELLOW:
-					return "preview_color_yellow.css";
-				case LIGHTYELLOW:
-					return "preview_color_light_yellow.css";
-				case GREY:
-					return "preview_color_grey.css";
-				case LIGHTGREY:
-					return "preview_color_light_grey.css";
-				default:
-					throw new InternalError();
-			}
+		BLUE("preview_color_blue.css"),
+		LIGHTBLUE("preview_color_light_blue.css"),
+		GREEN("preview_color_green.css"),
+		LIGHTGREEN("preview_color_light_green.css"),
+		RED("preview_color_red.css"),
+		LIGHTRED("preview_color_light_red.css"),
+		YELLOW("preview_color_yellow.css"),
+		LIGHTYELLOW("preview_color_light_yellow.css"),
+		GREY("preview_color_grey.css"),
+		LIGHTGREY("preview_color_light_grey.css");
+
+		public String cssfile;
```

Comment:
This is actually risky, because it can be publicly modified.  public *final* might be doable, but even that is against our current standards enforced by checkstyle.  We should have an accessor (getter) method for cssfile based on our current code standards.  (Target is get all fields to private if we can)